.github/workflows/changelog.yaml

```diff
@@ -0,0 +1,15 @@
+name: 'Check changelog for adhereance to keep-a-changelog'
+
+on:
+  workflow_dispatch:
+  pull_request:
+    paths:
+      - 'CHANGELOG.md'
+
+jobs:
+  lint:
+    steps:
+      - uses: jbangdev/jbang-action@v0.110.1
+      with:
+        script: heylogs@nbbrd
+        scriptargs: "lint CHANGELOG.md"
```

Comment:
Is this the right command? I tried using the CLI tool installed with `brew` but the subcommand there is `check`